src/screens/Customers/AddCustomer.tsx

```diff
@@ -191,21 +194,21 @@ function AddCustomer(props: AddCustomerProps) {
                     formik.setFieldValue('meterNumber', '')
                   }
                 }}
-                label={'Meter Type'}
+                label={intl(words.x_type, words.meter)}
```

Comment:
I think the `formik.values.selectedMeterType` also needs to be `intl(...)` since it's one of 3 different types of meters (`No Meter`, `Smart Meter`, `Analog Meter`) 

I'm also not sure how `value` works for `input` for a select display. Not sure if it needs to be the same value as one of the `MenuItem` values (which it wouldn't be if we translated it) or if the `value` just determines what is displayed. If it just determines what's displayed, we probably don't need to add `intl` around `formik.values.selectedMeterType`. 

You could test by wrapping `formik.values.selecetedMeterType` in an `intl`, putting some random translation in for it, hardcoding so that it uses the "my" translations and seeing if the record still correctly works (there's probably an easier way to test this core concept so feel free to do some derivative of this).

Reply:
Gotcha. I used `formik.setFieldValue('selectedMeterType', intl(meterType))` on line 190, and the behavior works as you described!

Reply:
Resolved!

Reply:
Wait I don't think that this works. If I change the translated value of "Smart Meter" to "Smart Meterssss" (simulating a burmese translation) in `resource.js` then the selection won't work. We might need to specially map the burmese --> english translations both ways, so we can display it in burmese but send the english variant to airtable so the record is correctly populated. Let's talk about this after team meeting so we can get a solution quickly!

Reply:
@julianrkung The issue should be resolved now with my most recent commit! 

### Adding a Customer
Now in `AddCustomer.tsx`, we can select the following (pretend we're in Burmese and `Smart Metersss` is Burmese for `Smart Meter`). We know this works because in the following screenshot, the tariffs in `getTariffs()` remained the same. 

![Screen Shot 2021-05-01 at 12 13 03 AM](https://user-images.githubusercontent.com/44735047/116774535-10079e00-aa12-11eb-9e1a-5b3f649c44a3.png)

Furthermore, after creating a transaction, we get the following in `meepanyar-node`. 

<img width="405" alt="Screen Shot 2021-05-01 at 12 14 48 AM" src="https://user-images.githubusercontent.com/44735047/116774561-3fb6a600-aa12-11eb-82f1-39240d150b57.png">


### Editing a Customer
Furthermore, `EditCustomer.tsx` seems to function as desired. I can successfully select a new meter while editing a user.
 
![Screen Shot 2021-05-01 at 12 15 47 AM](https://user-images.githubusercontent.com/44735047/116774575-6379ec00-aa12-11eb-9202-37b501ade236.png)

Then, upon moving to the next page, I properly get the tariff plan options from `getTariff()`. 

![Screen Shot 2021-05-01 at 12 16 41 AM](https://user-images.githubusercontent.com/44735047/116774597-83a9ab00-aa12-11eb-8b84-5cfd54f51efd.png)

Finally, I get the following message in my terminal indicating that the user has been authenticated and successfully updated (as far as I can tell). 

<img width="564" alt="Screen Shot 2021-05-01 at 12 17 15 AM" src="https://user-images.githubusercontent.com/44735047/116774613-97eda800-aa12-11eb-87c4-4b7c1d854e7d.png">

Let me know how this looks to you!